config/testgrids/kubernetes/sig-multicluster/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- JeremyOT
+- pmorie
+- lauralorenz
+- skitt
+
```

Comment:
This list is based on https://github.com/kubernetes/test-infra/blob/f8f44cacb522ad84e7e2fcc07d6fe4f9e11d1149/config/jobs/kubernetes-sigs/mcs-api/OWNERS

I omitted myself from here since it seems like more of a SIG-level thing. I don't think I would need to make any more changes in this directory to keep iterating on the MCS test configs.